styles.css

```diff
@@ -0,0 +1,88 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  height: 718px;
+  width: 375px;
+  background-color: black;
+}
+
+.bg {
+  display: flex;
+  flex-direction: column;
+  background-image: url("logos/header-shapes\ mobile@2x\ \(1\).jpeg");
+  height: 718px;
+  width: 375px;
```

Comment:
- [ Optional] You should remove the fixed width and height in pixels here. **NB:**  You can use Google Chrome developer tools if you want to see a mobile version while developing. Please check [this](https://www.youtube.com/watch?v=x4q86IjJFag) youtube video from Traversy Media on how to use developer tools.